mfthomps/CyberCIEGE-port-to-Unity3d
# Merge Request for Issue/286 - Help Tip revamp

Closes #286 

To test:
- Try to drag scrollbars on the Component View (after selecting a computer) while the help tip is up and make sure it works correctly
- Make sure UI highlights correctly on mouseover while help tip is up
- Make sure help tip goes away when the user clicks without dragging the mouse
